pkgs/development/python-modules/easysnmp/default.nix

```diff
@@ -0,0 +1,72 @@
+{ stdenv
+, lib
+, pkgs
+, buildPythonPackage
+, pythonOlder
+, fetchFromGitHub
+
+, python
+, pytest
+, pytestcov
+, pytest-flake8
+, pytest-sugar
+, termcolor
+, ipython
+}:
+
+buildPythonPackage rec {
+  pname = "easysnmp";
+  version = "0.2.5";
+
+  # See https://github.com/kamakazikamikaze/easysnmp/issues/108
+  disabled = !(pythonOlder "3.7");
+
+  src = fetchFromGitHub {
+    owner = "kamakazikamikaze";
+    repo = pname;
+    rev = version;
+    sha256 = "1si9iyxqj6z22jzn6m93lwpinsqn20lix2py3jm3g3fmwawkd735";
+  };
+
+  checkInputs = [
+    pytest
+    pytestcov
+    pytest-flake8
+    pytest-sugar
+    termcolor
+    pkgs.net_snmp
+  ];
+
+  buildInputs = [
+    pkgs.net_snmp
+    pkgs.openssl
+    pkgs.gcc
```

Comment:
I think gcc should be included as a result of this being built with stdenv anyway, not 100% sure though. If this builds without it, do remove it.

Reply:
for darwin it's clang. However, more importantly, it should be inherited from stdenv.cc. This line should be unnecessary